tests/testScripts/test_auth_user_can_explore_more_events.py

```diff
@@ -0,0 +1,22 @@
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
rather good functionality but not in this case. Better use pytest hooks here, if you want

Reply:
Tried to solve how to implement pytest hooks in this case, but it`s a little bit complicated for me